unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/LoadWrapper/LoadWrapper_GLTF.cs

```diff
@@ -22,7 +23,11 @@ public override void Load(string targetUrl, Action<LoadWrapper> OnSuccess, Actio
 
             alreadyLoaded = false;
             Assert.IsFalse(string.IsNullOrEmpty(targetUrl), "url is null!!");
-            loadHelper = new RendereableAssetLoadHelper(this.entity.scene.contentProvider, entity.scene.sceneData.baseUrlBundles);
+
+            if (contentProvider == null)
+                contentProvider = this.entity.scene.contentProvider;
+
+            loadHelper = new RendereableAssetLoadHelper(contentProvider, entity.scene.sceneData.baseUrlBundles);
```

Comment:
```suggestion
            if (customContentProvider == null)
                loadHelper = new RendereableAssetLoadHelper(this.entity.scene.contentProvider, entity.scene.sceneData.baseUrlBundles)
            else
                loadHelper = new RendereableAssetLoadHelper(customContentProvider , entity.scene.sceneData.baseUrlBundles);
```

I think it's more clear this way.